Sources/Meta/Models/Branch.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftSyntax
+
+public class Branch {
```

Comment:
I think `Branch` does not convey the meaning this class has. What about `NestedDeclaration`?

Reply:
I love it! :)